### DISCUSSION TITLE
Misleading Documentation

### DISCUSSION THREAD

**Discussion by nerisonpitogo** [dedoc/scramble#529] [General]
This is what is stated in the documentation.

https://scramble.dedoc.co/developers/how-it-works#putting-it-all-together

"All the endpoints (operations) are sorted alphabetically by their paths. Schema definitions are also sorted alphabetically by their names."

Spent hours on how to organize my endpoint to group them using @tags and arrange them the way I want. Then later I found out, in my case, they are arranged according to their arrangement in the api.php - top to bottom not alphabetically.

**Comment by romalytvynenko** [comment]
@nerisonpitogo 

I've tried to reproduce. As you can see, it is alphabetically sorted. Ensure you use the latest version, or provide a reproduction repository in case it still doesn't work.

<img width="509" alt="image" src="https://github.com/user-attachments/assets/e93d1ff4-2dba-492e-ac09-d4f09c83ae02">
<img width="417" alt="image" src="https://github.com/user-attachments/assets/9182b44c-9785-474a-b126-850eaaca0482">
<img width="362" alt="image" src="https://github.com/user-attachments/assets/02e8b071-c567-4b5a-8900-dae32be89b5b">
<img width="321" alt="image" src="https://github.com/user-attachments/assets/6881f27b-d005-40cf-9112-004995a0ada4">

**Comment by nerisonpitogo** [comment]
OMG. You are right. Seems like it's me who is misleading. I am referring to the order of the lists under the endpoint.  Is there a way to arrange this list aside from the order they are written in the api.php? These four endpoints belong to the `@tags AUTH`. 

![image](https://github.com/user-attachments/assets/abb1a364-139d-4cc1-b160-42fbc12387a1)
![image](https://github.com/user-attachments/assets/b5c801b5-7722-4613-b18a-3ec232aae7d4)

Rearranged.

![image](https://github.com/user-attachments/assets/4db102b2-624d-4742-b157-9e5819e52ac3)
![image](https://github.com/user-attachments/assets/f3827ab4-305c-4349-80e3-fbcab0ef07d7)

**Comment by romalytvynenko** [comment]
@nerisonpitogo Here I'd need your help! Can you tell me how are you trying to sort endpoints in your docs? And why? 

One one hand, I thing alphabetical sort is predictable. On the other hand, what if you're trying to "tell a story" with your docs. I mean that you may want to introduce your users to some concepts of your API in some order.

**Comment by nerisongotham** [comment]
I want it to be alphabetically arranged also the same with the endpoints so that it will easily be found. 

I want it to be.

1. Login a User
2. Register a new user via email
3. Send a Password Reset Email
4. Verify a User's email address

The reason behind this is that I am making a libraries endpoints. So under Libraries, there will be a lot of routes/endpoints from multiple CRUDs. Example Library for Rate Types, Library for Services.

But I just realized that putting them in separate endpoints is a better approach.
![image](https://github.com/user-attachments/assets/81cfd25b-e95e-43c4-9704-17895a2d5402)

rather than.

![image](https://github.com/user-attachments/assets/edcbb962-b8f4-42a5-85cc-d60978177241)

**Comment by nerisonpitogo** [comment]
Just realized im using different account onmy comment.
